panther_manager/plugins/decorator/tick_after_timeout_node.cpp

```diff
@@ -0,0 +1,49 @@
+#include <panther_manager/plugins/decorator/tick_after_timeout_node.hpp>
+
+#include <string>
+
+#include <behaviortree_cpp/basic_types.h>
+#include <behaviortree_cpp/tree_node.h>
```

Comment:
```suggestion
#include <behaviortree_cpp/tree_node.h>

#include <ros/console.h>
#include <ros/duration.h>
#include <ros/time.h>

```

Reply:
`#include <ros/console.h>` turned out to be a bit too far-fetched comment. This is not required here